### PR TITLE
Method to load installed packages in JSON format

### PR DESCRIPTION
**Pull request by yeexinc**
### Purpose

This PR adds a method in `LibraryViewController` that returns the user's installed packages in JSON.

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@sharadkjaiswal

### CODE REVIEW

**Comment by yeexinc** [comment]
@sharadkjaiswal - I've updated the method to obtain the installed packages. They can be passed to the PMUI using `getInstalledPackagesJSON()` through `TabControlConfig`.

Here's the PR for the PMUI to handle the new attributes: https://git.autodesk.com/Dynamo/PackageManagerUI/pull/28

Thanks!

File: src/LibraryViewExtension/LibraryViewController.cs [hunk 1]
```diff
@@ -201,5 +202,35 @@ private void OnDescriptionViewLoaded(object sender, System.Windows.RoutedEventAr
             var browser = view.Browser;
             browser.ConsoleMessage += OnBrowserConsoleMessage;
         }
+
+        public string GetInstalledPackagesJSON()
+        {
+            string pkg = null;
+            dynamoWindow.Dispatcher.Invoke(new Action(() => pkg = LoadInstalledPackagesJSON()));
```
Comment (sharadkjaiswal) [inline src/LibraryViewExtension/LibraryViewController.cs]: We need to use dispatcher only for UI actions, in this case, you don't need to use dispatcher.  Only one method should be good, you can merge the other method with this.

**Comment by yeexinc** [comment]
Hi @sharadkjaiswal , I've made the changes as discussed. Thanks for the review!

**Comment by sharadkjaiswal** [comment]
Thanks @yeexinc, I am merging this for now. Please create another PR to have a method to check if a package is already installed or not.

**Pull request merged**